source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -174,17 +176,17 @@ const SummaryList: React.FC<Props> = ({
           values.forEach((v, index) => {
             listItems.push(
```

Comment:
Out of this scope but this whole filter => forEach => forEach is really hard to read. These types of chained code blocks should be avoided in order to increase readability in my opinion.